assets/js/components/ExecutionResults/CheckResultOutline.jsx

```diff
@@ -8,21 +8,29 @@ const TARGET_CLUSTER = 'TARGET_CLUSTER';
 const isExpect = ({ type }) => type === EXPECT;
 const isExpectSame = ({ type }) => type === EXPECT_SAME;
 
-function CheckResultOutline({
-  checkID,
-  agentsCheckResults,
-  expectationResults,
-  clusterName,
-}) {
-  const expectExpectationsCount = expectationResults.filter(isExpect).length;
-  const expectResults = agentsCheckResults.map((agentCheckResult) => {
+const expectExpectations = (expectationList) =>
+  expectationList.filter(isExpect);
```

Comment:
We definitely could inline this (and other functions), however I am afraid that the code would become clunky to read (saying because that's where I started from before extracting functions :smile: )

What about a different naming?
`expectExpectations` -> `getExpectExpectations`|`getExpectExpectationsFromList`, and similarly for `expectSameExpectations`? Suggestions?

Reply:
Maybe `getExpectStatements`? I'd like to avoid `expectExpectation` :smile: